docs/next/modules/en/pages/getting-started/create-first-cluster/using_kubectl.adoc

```diff
@@ -21,6 +21,7 @@ export CLUSTER_NAME=cluster1
 export CONTROL_PLANE_MACHINE_COUNT=1
 export WORKER_MACHINE_COUNT=1
 export KUBERNETES_VERSION=v1.30.0
+export NAMESPACE=capi-cluster1
```

Comment:
Is this name intentional or a typo? Asking because in all other changes the namespace is called `capi-clusters` while only in this file it's called `capi-cluster1`.

Reply:
fixed